.github/workflows/ci.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     env:
       AWS_DEFAULT_REGION: us-east-1
       AWS_BACKUP_REGION: eu-central-1
+      GOOGLE_PROJECT_ID: test-project
```

Comment:
Is this a correct value? Seems like a placeholder

Reply:
the value could be anything afaik